src/log.js

```diff
@@ -195,16 +204,17 @@ export class Log {
       return LogLevel.INFO;
     }
 
-    return this.defaultLevelWithFunc_();
+    return this.defaultLevelWithFunc_(win);
   }
 
   /**
+   * @param {!Window} win
    * @return {!LogLevel}
    * @private
    */
-  defaultLevelWithFunc_() {
+  defaultLevelWithFunc_(win) {
     // Delegate to the specific resolver.
-    return this.levelFunc_(getMode().log, getMode().development);
+    return this.levelFunc_(logHashParam(win), getMode().development);
```

Comment:
I believe this is incorrect. Previously we always used the global `window` (the root AMP doc) and checked it for the hash. The global window is constant (it's per script evaluation), even when we create a log constructor for the embed window. That means the embed iframe doesn't need the `#log=foo` hash, because only the root window's hash matters.

With this change, we'll be using the embed iframe's window when we create the embed iframe's log. Now the embed iframe needs the `#log` hash, too.

```suggestion
    return this.levelFunc_(logHashParam(), getMode().development);
```